en/docs/develop/streaming-apps/working-with-the-design-view.md

```diff
@@ -1312,9 +1312,9 @@ connections between Siddhi components. A connection is defined by
 drawing an arrow from one component to another by dragging the cursor as
 demonstrated below.
 
-![Connecting Siddhi components]({{base_path}}/assets/img/streaming/working-with-the design-view/connect-siddhi-components.gif)
+![Connecting Siddhi components](/assets/img/streaming/working-with-the design-view/Connect-Siddhi-Components.gif)              ]
```

Comment:
We don't need the closing square bracket at the end.

Reply:
fixed with [33b1850](https://github.com/wso2/docs-apim/pull/7321/commits/33b1850d15ef4f967024552b40b35169ff6bc33c)